packages/next/package.json

```diff
@@ -107,12 +107,26 @@
     "webpack": "4.44.1",
     "webpack-sources": "1.4.3"
   },
+  "optionalDependencies": {
+    "sharp": "0.26.2"
```

Comment:
This appears to have inadvertently downgraded `sharp`!

Reply:
Ops, missed that in the merge conflict, fixed